Model/Client.php

```diff
@@ -17,6 +17,8 @@
 
 namespace Taxjar\SalesTax\Model;
 
+use Laminas\Http\Client as LaminasClient;
+use Laminas\Http\Request as LaminasRequest;
```

Comment:
Why not use Magento Framework classes? So when Magento decides to replace Laminas with smth. else we remain compatible.
```php
use \Magento\Framework\HTTP\LaminasClient
use \Magento\Framework\App\Request\Http
```